### PR TITLE
Use common helper for getting command line arguments.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: src/python/tests/core/bot/testcase_manager_test.py [hunk 1]
```diff
@@ -807,3 +807,35 @@ def test_target_not_found(self):
...
+    self.assertEqual('arg1 arg2',
+                     testcase_manager.get_command_line_flags('/fuzz-testcase'))
+
+  def test_only_args(self):
+    """Test both APP_ARGS and additional args."""
+    os.environ['APP_ARGS'] = 'arg1'
+    self.assertEqual('arg1',
+                     testcase_manager.get_command_line_flags('/fuzz-testcase'))
+
+  def test_only_additional(self):
```
Comment (inferno-chromium) [inline src/python/tests/core/bot/testcase_manager_test.py]: nit: add test on both APP_ARGS not set and no flags file.

Comment (oliverchang) [reply src/python/tests/core/bot/testcase_manager_test.py]: done!

**Pull request merged**